packages/office-ui-fabric-react/src/components/ChoiceGroup/ChoiceGroupOption/ChoiceGroupOption.base.tsx

```diff
@@ -49,6 +50,7 @@ export class ChoiceGroupOptionBase extends BaseComponent<IChoiceGroupOptionProps
       <div className={this._classNames.root}>
         <div className={this._classNames.choiceFieldWrapper}>
           <input
+            aria-label={ariaLabel || this.props.text}
```

Comment:
I think this was mentioned, but I'm not sure that we need aria-label to default to `this.props.text` given that we have the label rendering 
```
<span id={props.labelId} className="ms-ChoiceFieldLabel">
  {props.text}
</span>
```

I'm also now noticing that there doesn't seem to be an `aria-owns` for the label, even though that attribute is mentioned in the comments of `labelId`. 

> /**
>   \* DOM id to tag the ChoiceGroup label with, for reference.
>   \* Should be used for 'aria-owns' and other such uses, rather than direct reference for programmatic purposes.
>  */

Looks like we use `htmlFor` instead:

https://github.com/OfficeDev/office-ui-fabric-react/blob/ba457b8bb24332cdd51e92aaadf2dd9c66582892/packages/office-ui-fabric-react/src/components/ChoiceGroup/ChoiceGroupOption/ChoiceGroupOption.base.tsx#L98

Reply:
If we don't set  `aria-label` default to `this.props.text`, we will have problem in accessibility. Please  look at the code below.
`<input type="radio" id="A" aria-label=""></input>`
`<label for="A">option A</label>`

If we have `aria-label` even though it's null string, `label` will no longer be read from screen reader.
I have a quick test in code pen, see [here](https://codepen.io/qianl/pen/bmQQoB).

Label 'Option A' will not be read when focus on the radio input. Screen reader will only say 'non selected, radio button, one of one'. (browser: Edge, screen reader: Narrator)

Reply:
If you set the `aria-label` attribute to undefined in the React code, it will not be rendered and will not interfere with the other label. React will ignore all attributes that have undefined as their value. So the conditional

```
aria-label={ariaLabel ? ariaLabel : undefined}
```
 should work.